atmosphere/settings/local.py.j2

```diff
@@ -238,6 +233,9 @@ DATABASES = {
 SERVER_NAME = SERVER_URL.replace('https://','')
 EMAIL_SUBJECT_PREFIX = "[Django - %s] " % (SERVER_NAME,)
 
+# Sends all emails
+ATMO_DAEMON = ("Atmosphere Admin", "atmo-admin@{}".format(SERVER_NAME))
+
```

Comment:
This is defined in two places, should probably move into __init__.py. ideally we only have one settings file!

Reply:
I think it needs to be in local.py because it uses the SERVER_NAME that is defined here

Reply:
Okay thats fine